src/vm.rs

```diff
@@ -674,7 +673,7 @@ impl<'a, E: UserDefinedError> EbpfVm<'a, E> {
                 _ => return Err(EbpfError::UnsupportedInstruction(pc + ebpf::ELF_INSN_DUMP_OFFSET)),
             }
             if self.last_insn_count >= remaining_insn_count {
-                return Err(EbpfError::ExceededMaxInstructions(pc + 1 + ebpf::ELF_INSN_DUMP_OFFSET, self.total_insn_count));
+                return Err(EbpfError::ExceededMaxInstructions(pc + 1 + ebpf::ELF_INSN_DUMP_OFFSET));
```

Comment:
No longer reporting the instruction cap?

Reply:
See commit message:
"Removes the `total_insn_count` field from `ExceededMaxInstructions` as it was wrong
and the correct value can now be retrieved from `EbpfVm::get_total_instruction_count()`."
Or you could also ask your instruction meter before running the program, alternatively.

Reply:
I'm thinking from a debugging perspective, it's helpful when the error log show the max instructions hit